applications/OptimizationApplication/python_scripts/execution_policies/execution_policy_wrapper.py

```diff
@@ -0,0 +1,86 @@
+# ==============================================================================
```

Comment:
I suggest renaming the file and class to AnalysisExecutionPolicy.... to be more specific and consistent with other classes

Reply:
This class does not have an `ExecutionPolicy`, it just wraps around defined `ExecutionPolicy` such that, logging, pre, post operations are done automatically.